src/index.html

```diff
@@ -22,6 +23,367 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="/">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="header__logo-img"
+          />
+        </a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="nav-list">
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <section class="catalog">
+        <article
+          class="catalog__card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="monitor"
```

Comment:
better to provide more describeful names for the `alt` attribute, such as `APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)`

make sure to fix it everywhere in your code, not only here 
you might wanna use `ctrl + f` for this or `alt + d` combinations, to make it simpler